packages/office-ui-fabric-react/src/components/Calendar/Calendar.types.ts

```diff
@@ -16,6 +16,9 @@ export interface ICalendarProps extends React.Props<Calendar> {
    */
   componentRef?: (component: ICalendar | null) => void;
 
+  /** Optional class name to add to the root element. */
```

Comment:
>/** [](start = 2, length = 3)

Please make the comment to be multi line. Like 

/**
 * blah blah blah...
 */